softlayer/resource_softlayer_vlan.go

```diff
@@ -110,6 +111,10 @@ func resourceSoftLayerVlan() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 						},
+						"subnet_size": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
```

Comment:
I have a problem with this being a required property all of a sudden. Could it be made optional? Can there be a default?